charts/k8ssandra/values.yaml

```diff
@@ -103,11 +103,28 @@ stargate:
 
 # Configuration regarding configuration of repair services
 repair:
-  # Reaper repair service and UI
+  # reaper Configures Reaper which schedules and orchestrates repairs. Note that in general
+  # repairs should be performed within every gc_grace_seconds time. Reaper dramatically
+  # improves and simplifies running and managing repairs over doing so with nodetool.
   reaper:
-    autoschedule: false
+    # enabled Enables or Disables the Reaper deployment. Note that Reaper uses Cassandra's
+    # JMX APIs to perform repairs. When Reaper is enabled, Cassandra will also be
+    # configured to allow remote JMX access. JMX authentication will be configured in
+    # Cassandra with credentials only created for Reaper in order to limit access.
     enabled: true
 
+    # image Configures the Reaper image to use.
+    image:
+      # repository Specifies from where to pull the image.
+      repository: docker.io/thelastpickle/cassandra-reaper
+      # tag The image tag to pull.
+      tag: 2.1.3
```

Comment:
If Medusa has configurable pullPolicy, should Reaper also for consistency?

Reply:
We need to add it in the Reaper CRD. I created a ticket for this last night.